gateway-service-definitions/src/main/resources/services/atlas/2.1.0/rewrite.xml

```diff
@@ -32,14 +32,20 @@
         <rewrite template="{$frontend[path]}/atlas/"/>
     </rule>
 
-    <rule dir="OUT" name="ATLAS/atlas/outbound/login/headers/location">
-        <match pattern="*://*:*/login.jsp"/>
-        <rewrite template="{$frontend[url]}/atlas/login.jsp"/>
+    <rule dir="OUT" name="ATLAS/atlas/outbound/login/headers/location" flow="OR">
+        <match pattern="*://*:*/login.jsp">
+            <rewrite template="https://{$frontend[addr]}/gateway/knoxsso/knoxauth/logout.jsp?originalUrl={$postfix[url,/atlas/]}" />
```

Comment:
Here we are assuming that the cluster name is gateway which will not be the case always.